internal/driver/glfw/testdata/windows_hover_object.xml

```diff
@@ -1,36 +1,35 @@
-<canvas size="200x300">
+<canvas size="50x45">
```

Comment:
I'm not sure that this change is an expected outcome? A lot has changed in this test output which doesn't seem right.